isis/src/control/objs/BundleUtilities/BundleObservation.cpp

```diff
@@ -783,6 +783,761 @@ namespace Isis {
+
+        }//end inner-for
+
+      }// end outer-for
+
+     } //end outer-outer for
+
+     for (int i = 0; i < nPointingCoefficients; i++) {
+        coefRA[i]*=RAD2DEG;
+        coefDEC[i]*=RAD2DEG;
+        coefTWI[i]*=RAD2DEG;
+     }
+
+     finalParameterValues.push_back(coefRA);
+     finalParameterValues.push_back(coefDEC);
+     finalParameterValues.push_back(coefTWI);
+
+
+
+     for (int i = 0; i < imagePointingParams.size(); i++) {
+           if (nPointingCoefficients > 0) {
+           for (int j = 0; j < nPointingCoefficients;j++) {
+             if (j == 0) {
+               parameterNamesList.append(str.arg(imagePointingParams[i]).arg("dd"));
+               correctionUnitList.append("dd");
+             } //end inner-if
+
+             else {
+               switch(j) {
+               case 1:
+                 parameterNamesList.append( str.arg("     ").arg("dd/s") );
+                 correctionUnitList.append("dd/s");
+                 break;
+               case 2:
+                 parameterNamesList.append( str.arg("     ").arg("dd/s^2") );
+                 correctionUnitList.append("dd/s^2");
+                 break;
+               case 3:
+                 parameterNamesList.append( str.arg("     ").arg("dd/s^3") );
+                 correctionUnitList.append("dd/s^3");
+                 break;
+               default:
+                 parameterNamesList.append(str.arg("     ").arg("dd/s^"+toString(j) ) );
+                 correctionUnitList.append("dd/s^"+toString(j));
+             } //end switch-case
+           }  //end inner-else
+
+         }//end inner-for
+
+       }// end outer-for
+
+      } //end outer-outer for
+
+
+     std::vector<double> finalParameterValuesSingleVector;
+
+     for (unsigned int i = 0;i < finalParameterValues.size();i ++) {
+       for (unsigned int j = 0; j < finalParameterValues[i].size(); j++) {
+          finalParameterValuesSingleVector.push_back(finalParameterValues[i][j]);
+       }
+     }//end outer-for
+
+    // Save the list of parameter names we've accumulated above
+    m_parameterNamesList = parameterNamesList;
+
+    // Set up default values when we are using default position
+    QString sigma = "N/A";
+    QString adjustedSigma = "N/A";
+    double correction = 0.0;
+
+      // position parameters
+      for (int i = 0; i < nPositionParameters; i++) {
+        // If not using the default position, we can correctly access sigmas and corrections
+        // members
+        if (!useDefaultPosition) {
+          correction = m_corrections(i);
+          adjustedSigma = QString::number(m_adjustedSigmas[i], 'f', 8);
+          sigma = ( IsSpecial(m_aprioriSigmas[i]) ? "FREE" : toString(m_aprioriSigmas[i], 8) );
+        }
+        if (errorPropagation) {
+
+          sprintf(buf,"%-*s",10,parameterNamesList.at(i).toStdString().c_str() );
+          fpOut << buf;
+          sprintf(buf,"%15.2f",finalParameterValuesSingleVector[i] - correction);
```

Comment:
Don't forget that this parameter name may be changed, but should still be accessed the same